src/ConfigAggregator.php

```diff
@@ -60,19 +60,21 @@ class ConfigAggregator
      * @param array $postProcessors Array of processors. These may be callables, or
      *     string values representing classes that act as processors. If the
      *     latter, they must be instantiable without constructor arguments.
+     * @param int $mode File mode to set on cached config
```

Comment:
I think this extra note is warranted here as using decimal value is a rather subtle mistake.
```suggestion
     * @param int $mode File mode to set on cached config.
     *     Note that mode is an octal value, so to ensure the expected operation,
     *     you need to prefix mode with a zero (eg 0644)
     *     {@see http://php.net/manual/en/function.chmod.php }
```